Frontend/CO.CDP.OrganisationApp/Pages/Forms/FormsAnswerSetRemoveConfirmation.cshtml.cs

```diff
@@ -24,7 +25,7 @@ public class FormsAnswerSetRemoveConfirmationModel(
     public Guid AnswerSetId { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Please select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Global_SelectAnOption))]
```

Comment:
Can we go with the standard `ErrorMessageResourceName` / `ErrorMessageResourceType` pairing please as found elsewhere. The plain `ErrorMessage` option is supposed to be reserved for untranslated text.

Reply:
Ok